### DISCUSSION TITLE
Why is there no support for NativeAd?

### DISCUSSION THREAD

**Discussion by kurtgn** [invertase/react-native-firebase#4285] [Q&A] [answered]
This awesome library offers three Admob unit types - `Interstitial`, `RewardedAd` and `BannerAd`, but why is there no support for `NativeAd` ?

I can see in the docs that there is a base class called `MobileAd` which is supposed to be subclassed by `NativeAd`

https://rnfirebase.io/reference/admob/mobilead

but the codebase does not contain `NativeAd`. Why is that?

**Comment by mikehardy** [comment, marked as answer]
Probably because no one implemented it :-). Welcome! PRs happily accepted :-)

**Comment by kurtgn** [comment]
right, in a year maybe, I am a total newbie in Android for now :)
